web/src/App.svelte

```diff
@@ -259,6 +259,15 @@
 </div>
 
 <style>
+  :global(#app .left) {
+    width: 35%;
+    min-width: 400px;
+    max-width: 700px;
+  }
```

Comment:
Previously we had a little bit wider left column for CNT, I think to accommodate the three column LAD list, which makes sense.

But honestly, I think we could do a little wider in general, especially on narrower screens. 

I've also added some min/max widths that I think offer reasonably consistent feeling. All the particular values are up for debate, but I think having *some* kind of min/max is desirable.